### ISSUE TITLE
Allow Deco gasses with Helium and make it possible to choose from which depth to use a deco gas.

### ISSUE THREAD

**Issue by hansalves** [NeoTech-Software/Abysner#10]

**Comment by Rolf-Smit** [comment]
Helium can already be used, the app automatically chooses the best gas for the ascent. I need to double check what the exact rules for the automatic selection are.

I changed the title to something that reflects what's missing a bit better.

**Comment by pabdakine** [comment]
Wouldn't say "range", but a maximum depth overriding the standard/default 1.6 criterion. I often use this to fit within the constraints of available gas with a slight increase in total time.

For interface, I suggest changing the area that currently says "nitrox", etc. for the MOD override (as that's redundant with the numeric mix).

**Comment by Rolf-Smit** [comment]
@pabdakine yeah that is a fair point. Range is the wrong wording here, it should have been a 'fixed depth'.
